### ISSUE TITLE
Need information on sigstore-docs build

### ISSUE THREAD

**Issue by jonvnadelberg** [sigstore/docs#197]
**Question**

I'm trying to rearrange the files and folders in the docs.sigstore.dev website.  I want to add some top-level categories and rename the current categories.

My question is where does this information come from?   There must be a place where these categories are named.   I cannot find it in the github repository.   Does anyone know how this is set up?  I am thinking it has something to do with yarn, but do not know the location of the files.

This is regarding issue #191

**Comment by jonvnadelberg** [comment]
Hi @haydentherapper @ltagliaferri @smythp @olivekl 

If any of you know where this information is stored, I'd appreciate it, as I will have to be editing it.

**Comment by haydentherapper** [comment]
Titles I believe come from `menuTitle`, like https://github.com/sigstore/docs/blob/main/content/en/index.md?plain=1#L5. To group pages, you would create new folders, like how there is currently a folder for "cosign", "fulcio", etc.

I believe ordering is based on `position` - For example, https://github.com/sigstore/docs/blob/main/content/en/index.md?plain=1#L6, which would put Overview as first, and main concepts (https://github.com/sigstore/docs/blob/main/content/en/main-concepts.md?plain=1#L4) as second.

**Comment by jonvnadelberg** [comment]
> Titles I believe come from `menuTitle`, like https://github.com/sigstore/docs/blob/main/content/en/index.md?plain=1#L5. To group pages, you would create new folders, like how there is currently a folder for "cosign", "fulcio", etc.
> 
> I believe ordering is based on `position` - For example, https://github.com/sigstore/docs/blob/main/content/en/index.md?plain=1#L6, which would put Overview as first, and main concepts (https://github.com/sigstore/docs/blob/main/content/en/main-concepts.md?plain=1#L4) as second.

Thanks @haydentherapper i thought I made a new folder in github and added a file, but it didn't show up.   I'll try again.

**Comment by smythp** [comment]
I just played around with this and got new headings and files working. There's some fiddliness. Here are the steps I used:

1. Create a new folder, i.e. `newheading`
2. Put a file in it, i.e. `newfile.md`
3. In the newfile.md` file, add the YML, but be sure to set the category as the newly created folder. The title is what will show up on the left navbar as well as the title page.

```
---
title: "New Heading"
category: "newheading"
position: 1
---
```

As @haydentherapper  noted, the position metadata determines where the file appears relative to other files in that category. The confusing part here is that it seems like the new category and file don't appear on the left navbar unless you get the category metadata right.
